From 4fb0e0df110e13e5459028b9c4ad7c55be7c93e4 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Wed, 26 Sep 2018 21:18:06 +0300
Subject: [PATCH 03/14] mm-qcamera-daemon: fix denial

avc: denied { use } for path="anon_inode:dmabuf" dev="anon_inodefs"
     ino=11426 scontext=u:r:mm-qcamerad:s0
     tcontext=u:r:platform_app:s0:c512,c768 tclass=fd

Change-Id: I1de19511563236ee4a7a0bd395f711e1bcb189c4
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 common/mm-qcamerad.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/mm-qcamerad.te b/common/mm-qcamerad.te
index 70e14d1..99d20f9 100644
--- a/common/mm-qcamerad.te
+++ b/common/mm-qcamerad.te
@@ -40,7 +40,7 @@ allow mm-qcamerad self:process execmem;
 allow mm-qcamerad video_device:dir r_dir_perms;
 allow mm-qcamerad { gpu_device video_device sensors_device }:chr_file rw_file_perms;
 
-allow mm-qcamerad { surfaceflinger mediaserver cameraserver hal_camera untrusted_app_all priv_app }:fd use;
+allow mm-qcamerad { surfaceflinger mediaserver cameraserver hal_camera untrusted_app_all priv_app platform_app }:fd use;
 
 allow mm-qcamerad camera_data_file:dir w_dir_perms;
 #allow mm-qcamerad camera_data_file:sock_file { create unlink };
-- 
2.17.1

